docs/content/connectors/oceanbase-cdc.md

```diff
@@ -94,6 +94,12 @@ Flink SQL> SELECT * FROM orders;
 Connector Options
 ----------------
 
+The OceanBase CDC Connector contains some options for both sql and stream api as the following sheet. 
+
+*Note*: The connector supports two ways to specify the table list to listen to, and will get the union of the results when both way are used at the same time.
+1. Use `database-name` and `table-name` to match database and table names in regex. Can only be used in `initial` startup mode.
```

Comment:
Because the 'liboblog' which used to fetch change events log from OceanBase can only support fnmatch for now. If we want to use regex for change events, we must fetch all the log data firstly and do filter in the connector.

Reply:
Got it， Could we also add the background to the note?

Reply:
OK.